requirements.txt

```diff
@@ -4,23 +4,17 @@
 #
 #    pip-compile --output-file=requirements.txt requirements.in
 #
-asgiref==3.6.0
+asgiref==3.7.2
     # via django
-certifi==2022.12.7
+certifi==2023.5.7
     # via
     #   requests
     #   sentry-sdk
 cffi==1.15.1
     # via cryptography
 charset-normalizer==3.1.0
     # via requests
-coreapi==2.3.3
-    # via drf-yasg
-coreschema==0.0.4
-    # via
-    #   coreapi
-    #   drf-yasg
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?